### DISCUSSION TITLE
14.05 Agenda

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#284] [Weekly Meeting Agenda]
# Ideen 

- [x] Issue für die Vorlage zur Überarbeitung der bestehenden Issues und der Erstellung der Issues aus den User Stories
- [x] Architektur besprechen
- [ ] Aufteilung der Bereich jeder soll User Stories durchgehen und verbessern aber erst danach

# Task
| Task / Issue | Zuständig |
| ----------- | ----------- |
| #285  | |
| #286 | |
| #251 |  |
| #254 | |
| #250 ||
| #221 ||

# Questions
| Fragen / Ideen | Beantwortet | Antwort |
| ----------- | ----------- | ------ | 
| Sollen die Issues, die schon done sind, aber bei einer User Story verlinkt sind, auch überarbeitet/angepasst an die neuere Template werden? |  Ja/nein | Wird nach der Überarbeitung der anderen Issues nochmal besprochen |
| | | |
| | | |
| | | |
| | | |

# Task from last week
| Issue / Task | Zuständig | Erledigt ?|  Zeit |
| ----------- | ----------- | -------- | ----|
| #250  | @Kronprinz03  | To-Do | 230min |
| | | |
| | | |
| | | |

**Comment by two-Gee** [comment]
# Idee
 User Stories als "Oberissues" nehmen, um Dopplungen zu vermeiden
Anstatt nur ein Frontend und ein Backend Issue pro User Story zu machen, direkt detaillierte Issues erstellen und diese den User Stories zuordnen. 

### Zuordnung in User Stories: 
Frontend:
- Issue 1
- Issue 2

Backend: 
- Issue 1 
- Issue 2


# Vorschlag Issues Template: 

## Beschreibung 
(Genaue Beschreibung um was es geht und was in dem Issue umgesetzt werden soll. 
-> Möglichst implementierungsnah / technisch, **Dopplungen mit User Story vermeiden!**)
## Aufgaben 
(Beschreibung der Aufgaben als Task Liste, dabei genau auf Implementierungschritte eingehen)
## Zusätzliche Informationen 
### Abhängigkeiten
(Ggf. Abhängigkeiten zu anderen Issues verlinken)
### Mockup
(Bei Frontend: entsprechendes Figma Mockup einfügen)
### Architektur
(Bei Backend: entsprechende Architektur einfügen)

## User Story
Zugehörige User Story verlinken

## Sonstige ToDos 
(Punkt löschen vor Erstellen des Issues)
- Issue in entsprechender User Story hinzufügen  
- Label für Front oder Backend setzen 
- Label für Phase setzen
- Label für User Story
- zu Projekten hinzufügen
